src/locales/fr/pokemon-summary.ts

```diff
@@ -0,0 +1,17 @@
+import { SimpleTranslationEntries } from "#app/plugins/i18n";
+
+export const pokemonSummary: SimpleTranslationEntries = {
+  "pokemonInfo": "Info Pokémon",
+  "status": "Status",
+  "powerAccuracyCategory": "Puissance\nPrécision\nCatégorie",
+  "type": "Type",
+  "unknownTrainer": "Inconnu",
+  "ot": "DO",
```

Comment:
```suggestion
  "ot": "D.O.",
```
![image](https://github.com/pagefaultgames/pokerogue/assets/8146474/7490c23d-faa6-4bbd-b4f1-6922ecad5c48)